project_stock/models/project_task.py

```diff
@@ -72,12 +73,10 @@ class ProjectTask(models.Model):
         string="Move Analytic Account",
         help="Move created will be assigned to this analytic account",
     )
-    stock_analytic_tag_ids = fields.Many2many(
-        comodel_name="account.analytic.tag",
-        relation="account_analytic_tag_project_task_stock_rel",
-        column1="project_task_id",
-        column2="account_analytic_tag_id",
-        string="Move Analytic Tags",
+    stock_analytic_distribution = fields.Json(
+        "Move Analytic Tags",
+        copy=True,
+        readonly=False,
     )
```

Comment:
I'm not completely sure, but it is necessary to add a migration script to transform and keep the old data, right?

Reply:
Yes, this will require a migration script, although it's still not defined how to do it on core fields, and this will probably generates an openupgradelib method, so it can be left as is for now.